packages/@aws-cdk/aws-logs/lib/log-group.ts

```diff
@@ -169,6 +174,13 @@ abstract class LogGroupBase extends Resource implements ILogGroup {
     return this.grant(grantee, 'logs:CreateLogStream', 'logs:PutLogEvents');
   }
 
+  /**
+   * Give permissions to read and filter events from this log group
+   */
+  public grantRead(grantee: iam.IGrantable) {
+    return this.grant(grantee, 'logs:FilterLogEvents', 'logs:GetLogEvents', 'logs:GetLogRecord');
```

Comment:
[According to the docs](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_GetLogRecord.html), GetLogRecord is only used with a response from `GetQueryResults`. Any reason why you've included that?
```suggestion
    return this.grant(grantee, 'logs:FilterLogEvents', 'logs:GetLogEvents');
```

Reply:
We could probably throw a `DescribeLogGroups` and `DescribeLogStreams` in here
Also `GetLogGroupFields`